test/system/src/main/java/io/pravega/test/system/framework/services/docker/DockerBasedService.java

```diff
@@ -73,22 +73,44 @@ public String getID() {
     @Override
     public boolean isRunning() {
         boolean value = false;
-        int taskRunningCount = 0;
         try {
             //list the service with filter 'serviceName'
             Service.Criteria servicecriteria = Service.Criteria.builder().serviceName(serviceName).build();
             List<Service> serviceList = Exceptions.handleInterruptedCall(
                     () -> dockerClient.listServices(servicecriteria));
             log.info("Service list size {}", serviceList.size());
+            if (!serviceList.isEmpty()) {
+                long replicas = Exceptions.handleInterruptedCall(
+                        () -> dockerClient.inspectService(serviceList.get(0).id()).spec().mode().replicated().replicas());
+                log.info("Replicas {}", replicas);
+                if (replicas > 0 && isSynced() == true){
```

Comment:
You can replace `isSynced() == true` by just `isSynced()`.

Reply:
Fixed